static/schemas/obms/amt-obm-service.json

```diff
@@ -0,0 +1,29 @@
+{
+    "title": "amt-obm-service",
+    "definitions": {
```

Comment:
Why not put it under `defintions` rather than `properties`? My understanding is the `definitions` is something like `typedef` in C language, but `properties` is where you put properties that the object contains.